fili-core/src/test/resources/testApplicationConfig.properties

```diff
@@ -48,10 +48,21 @@ bard__druid_seg_loader_timer_delay = 60000
 # Don't start Druid dimension loading during tests
 bard__druid_dim_loader_timer_delay = 60000
 
-# Data Cache strategy, which is one of the following
-# 1. ETagOnly
-# 2. LocalSignatureOnly (cache V2, needs the above flag set as well)
-# 3. TtlOnly (cache V1)
+# Data Cache strategy, whose value is one of the following
+# 1. ETag
+#    This is the same thing as LocalSignature except that Etag optimizes to avoid the possibility of cache out-of-sync
+#    problem
+# 2. LocalSignature (cache V2, deprecated)
```

Comment:
I would make the point out that ETag relies on Druid to generate the signature instead of Fili.